packages/core/core/src/utils.js

```diff
@@ -235,3 +236,7 @@ export function toInternalSymbols<T: {|loc: ?SourceLocation|}>(
     ]),
   );
 }
+
+export function keyFromEnvOrOptionContentKey(contentKey: ContentKey): string {
+  return contentKey.slice(contentKey.indexOf(':') + 1);
```

Comment:
I think it'd be better to have an explicit function for both `env` and `option`, that way you don't need to search the string dynamically. For example, env nodes are always `contentKey.slice('env:'.length)` which saves the `indexOf` check. You could also co-locate them with the `nodeFrom*` functions so they can stay in sync.